docs/source/languages/orquesta.rst

```diff
@@ -107,16 +107,103 @@ execution, the task is evaluated for completion. If criteria for transition is m
+
+      parallel_task_3:
+        # ...
+        # Run immediately after setup_task, do NOT wait for barrier_task
+
+Will result in this execution graph:
+
+.. code-block:: none
+
+    =---- time (not to scale) ---->
+
+    setup_task --+
+                 |
+                 +------ parallel_task_1 --------------------------+
+                 |                                                 |
+                 +-- parallel_task_2 --+                           |
+                 |                     |                           |
+                 |                     +---- intermediate_task ----+
+                 |                                                 |
+                 |                                                 +-- barrier_task --+
+                 |                                                                    |
+                 +-- parallel_task_3 -------------------------------------------------+
+                                                                                      |
+                                                                                      +-- [finish]
 
 Conversely, if more than one tasks transition to the same task and ``join`` is **not** specified in
 the latter (i.e. the task named "log" in the example below), then the target task will be invoked
 immediately following the completion of the previous task. There will be multiple instances of the
 target task. In the workflow graph, each invocation of the target task will be its own branch with
 the inbound edge from the node of the previous task.
 
+In other words, if ``join: all`` was removed from the previous workflow, the ``barrier_task`` would
+be run two different times, resulting in this execution graph:
+
+.. code-block:: none
+
+    =---- time (not to scale) ---->
+
+    setup_task --+
+                 |
+                 +------ parallel_task_1 ------+
+                 |                             |
+                 |                             +-- barrier_task (1) ----------------------+
+                 |                                                                        |
+                 +-- parallel_task_2 --+                                                  |
+                 |                     |                                                  |
+                 |                     +---- intermediate_task ----+                      |
+                 |                                                 |                      |
+                 |                                                 +-- barrier_task (2) --+
+                 |                                                                        |
+                 +-- parallel_task_3 -----------------------------------------------------+
+                                                                                          |
+                                                                                          +-- [finish]
+
```

Comment:
Please remove `(i.e. the task named "log" in the example below)` in the paragraph above as this refers to a different example. Since you include an example here, let's remove the wrong reference here.